pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -400,6 +414,10 @@ func (c *Reconciler) reconcile(ctx context.Context, pr *v1alpha1.PipelineRun) er
 	}
 
 	for _, rprt := range pipelineState {
+		c.Logger.Infof("PipelineTask: %+v", rprt.PipelineTask)
+		c.Logger.Infof("PipelineTask.Params: %+v", rprt.PipelineTask.Params)
+		c.Logger.Infof("ResolvedTaskResources: %+v", rprt.ResolvedTaskResources)
+		c.Logger.Infof("ResolvedTaskResources.TaskSpec: %+v", rprt.ResolvedTaskResources.TaskSpec)
```

Comment:
This will generate quite a lot of log messages, is it useful to always have this?

Reply:
not at all :man_facepalming: I forgot to remove those.....